scripts/src/cowidev/vax/batch/latvia.py

```diff
@@ -5,22 +5,23 @@
 
 vaccine_mapping = {
     # "Ambirix": "",
-    "BBIBP-CorV": "Sinopharm/Beijing",
+    # "BBIBP-CorV": "Sinopharm/Beijing",
     "BBIBP-CorV(Sinopharm)": "Sinopharm/Beijing",
     "Comirnaty": "Pfizer/BioNTech",
-    "COMIRNATY": "Pfizer/BioNTech",
-    "CoronaVac": "Sinovac",
+    # "COMIRNATY": "Pfizer/BioNTech",
+    # "CoronaVac": "Sinovac",
     "CoronaVac(Sinovac)": "Sinovac",
-    "COVID-19 Vaccine Janssen": "Johnson&Johnson",
-    "COVID-19 Vaccine Moderna": "Moderna",
-    "Covishield": "Oxford/AstraZeneca",
+    # "COVID-19 Vaccine Janssen": "Johnson&Johnson",
+    # "COVID-19 Vaccine Moderna": "Moderna",
+    # "Covishield": "Oxford/AstraZeneca",
     "Covishield(ChAdOx1_nCoV-19)": "Oxford/AstraZeneca",
-    "Moderna COVID-19 vaccine": "Moderna",
+    "Jcovden": "Johnson&Johnson",
+    # "Moderna COVID-19 vaccine": "Moderna",
+    "Nuvaxovid": "Novavax",
     "Spikevax": "Moderna",
```

Comment:
The old vaccine names can be removed or left as is, and my current choice is to leave them in the comments.